plugins/transforms/selectvalues/src/test/java/org/apache/hop/pipeline/transforms/selectvalues/SelectValuesMetaInjectionTest.java

```diff
@@ -32,38 +33,59 @@ public class SelectValuesMetaInjectionTest extends BaseMetadataInjectionTest<Sel
 
   @Before
   public void setup() throws Exception {
-    setup(new SelectValuesMeta());
+    SelectValuesMeta selectValuesMeta = new SelectValuesMeta();
+    selectValuesMeta
+        .getSelectOption()
+        .setSelectFields(SelectValueMetaTestFactory.getSelectFields("v", "v2"));
+    setup(selectValuesMeta);
   }
 
   @Test
   public void test() throws Exception {
```

Comment:
This doesn't pass. I can't actually understand how it's building up the meta object, because apparently, it was empty, previously.
Now, with the new structure I get a IndexOutOfBoundException, which I can't fix without understanding what's the subject under test and how I can put it together.